commands/Moderation/unban.js

```diff
@@ -0,0 +1,30 @@
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['BAN_MEMBERS'],
+			runIn: ['text'],
+
+			description: 'Unbans a user id.',
+			usage: '<userid:string> [reason:string] [...]',
```

Comment:
userid should be a user, it uses client.users.fetch which uses the API and that allows you to pass any id that is on discord not just bots your bot can see.

Reply:
I was not aware of this.